miio/dreamevacuum_miot.py

```diff
@@ -5,11 +5,11 @@
 
 from .click_common import command, format_output
 from .miot_device import DeviceStatus as DeviceStatusContainer
-from .miot_device import MiotDevice, MiotMapping
+from .miot_device import MiotDevice
 
 _LOGGER = logging.getLogger(__name__)
 
-_MAPPING: MiotMapping = {
+_MAPPING = {
```

Comment:
Please do not remove the type hint.

Reply:
Actually I don't know why this part is changed, because I didn't change it myself. I will revert this.